plugin/src/main/scala/io.github.jonas.paradox.material.theme/ParadoxMaterialThemePlugin.scala

```diff
@@ -26,7 +26,7 @@ object ParadoxMaterialThemePlugin extends AutoPlugin {
     paradoxMaterialTheme / version :=
       Option(ParadoxPlugin.readProperty("paradox-material-theme.properties", "version"))
         .getOrElse(sys.error("Undefined paradox-material-theme version")),
-    paradoxTheme := Some("com.github.sbt" % "sbt-paradox-material-theme" % (paradoxMaterialTheme / version).value)
+    paradoxTheme := Some("com.github.sbt" % "paradox-material-theme" % (paradoxMaterialTheme / version).value)
```

Comment:
Sorry for being late here. So actually this was I was talking about in https://github.com/sbt/sbt-paradox-material-theme/pull/39/files#r1476823522

Reply:
Yup I realized that after, all good now!